proof/refine/RISCV64/TcbAcc_R.thy

```diff
@@ -3029,9 +3029,16 @@ lemmas no_fail_scActive[wp] =
 crunch inReleaseQueue
   for inv[wp]: P
 
+defs tcb_at'_asrt_def:
+  "tcb_at'_asrt \<equiv> \<lambda>tcbPtr s. tcb_at' tcbPtr s"
+
+declare tcb_at'_asrt_def[simp]
+
 defs sc_at'_asrt_def:
   "sc_at'_asrt \<equiv> \<lambda>scPtr s. sc_at' scPtr s"
 
+declare sc_at'_asrt_def[simp]
```

Comment:
It is a shame that the defs command doesn't take [simp] as a modifier, because the declare is a bit of an eyesore.

Reply:
The syntax of the command is now under our control, actually. We could probably make it take attributes if we wanted to.

Reply:
Do you think we should? Or make an issue? It is kinda rare.

Reply:
cheers